en/about/index.md

```diff
@@ -8,10 +8,15 @@ image: "/banner/aboutus.jpg"
 ---
 
 ## SOCIAL MISSION
-The club Walking Machine is before all a way to learn. Through their participation in the activities of the club, members acquire knowledge in the fields of engineering related to their. The nature of the competition in which the club participated, It has an important social mission: assistance to persons with reduced mobility. It is developing robots capable of performing simple household tasks that we want to improve the lot of our elders. On another level, the contribution of the youth club is just as important. With various events for the promotion of Robotics, an interest to continue in the area is found in young people in college and secondary schools.
+
+The Walking Machine club is, before everything, a way to learn. Its members gain knowledge through practice in the different fields of ingeeneering. The scientific club has also a social mission, to help citizen with reduced mobility, due to the nature of the competition in which it participates. By developping robots able to accomplish simple domestic tasks, we seek to make life easier for our elders. On another level, we also want to inspire youth, by showing up to diverse promotional events in highschool and colleges. 
```

Comment:
**Walking Machine is, before all, a learning experience.** Its members gain knowledge through practice in the different fields of **engineering**. **The nature of the competition which the club participates in has an important social mission: to assist people with reduced mobility.** 

Le reste est bon :)